### PR TITLE
feature: Added phpunit db setup tests

### PR DESCRIPTION
**Pull request by laf**
DO NOT DELETE THIS TEXT

#### Please note

> Please read this information carefully. You can run `./scripts/pre-commit.php` to check your code before submitting.

- [x] Have you signed the [Contributors agreement](http://docs.librenms.org/General/Contributing/) - please do NOT submit a pull request unless you have (signing the agreement in the same pull request is fine). Your commit message for signing the agreement must appear as per the docs.
- [x] Have you followed our [code guidelines?](http://docs.librenms.org/Developing/Code-Guidelines/)

I'm sure @murrant will rewrite this but it gets us validation at the moment that build-base will work

Doesn't use mysql strict at the moment as it will fail with this.

### CODE REVIEW

**Comment by laf** [comment]
Not sure why hhvm is failing here, especially seeing as it's set to allow failures.

**Comment by laf** [comment]
@librenms/reviewers Rebased and it's now all working, think murrant fixed something :)

I have tested php build-base.php and web installs still work off the back of this.

File: includes/init.php [hunk 1]
```diff
@@ -32,8 +32,10 @@
 chdir($install_dir);
 
 // Libraries
-require('Net/IPv4.php');
-require('Net/IPv6.php');
+if (!module_selected('phpunit', $init_modules)) {
```
Comment (laf) [inline includes/init.php]: I couldn't install them into the travis env so the test would fail.

Comment (murrant) [inline includes/init.php]: Odd, I don't remember running into that problem...

Comment (laf) [inline includes/init.php]: How did you do it before? We ask users to install them via pear or apt-get normally.

**Comment by laf** [comment]
Reverted back to not using net/ipvX modules.

**Comment by laf** [comment]
Yes I put the do test first so it's ready for other tests. What's up with it being first?

**Comment by murrant** [comment]
I changed around some things, so it wasn't done first.  I can put it back.  But right now, it creates a db, builds it up and then drops the database, so it is self-contained and no other tests rely on it.

If you want to build up a database for other tests to use, I don't think this is the right approach.

**Comment by laf** [comment]
I know overall it's not the right approach but we don't have a proper dbal to make use of things like in v2 so at present this is just a make do with what we have.

**Comment by murrant** [comment]
I'm happy with this PR as a way to test our sql schema files.  I think we should merge this as is and make changes later when we need the DB for another test.

**Comment by laf** [comment]
I have started to look at using a dbal and writing wrappers around our current calls to the dbal. Probably the easiest way to deal with this for now.

**Comment by murrant** [comment]
I looked at that awhile ago, then I ended up at a spot where I wanted full eloquent :(

Then, I decided it was better to focus energy elsewhere, like v2.

**Pull request merged**